components/ui/Modal.tsx

```diff
@@ -21,13 +21,11 @@ export const Modal: React.FC<ModalProps> = props => {
       <Column fill safe align="center">
         <Row elevation={props.headerElevation}>
           <View
-            style={{
-              flex: 1,
-              flexDirection: 'row',
-              alignItems: 'center',
-              marginHorizontal: 18,
-              marginVertical: 8,
-            }}>
+            style={
+              props.progressModalStyle
+                ? Theme.ModalStyles.progressingModal
+                : Theme.ModalStyles.defaultModal
+            }>
```

Comment:
Send style as prop. Modal component should not bind with progressing modal or any other component
We can send props as
customStyle and default it to Theme.ModalStyles.defaultModal.

Reply:
resolved